cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/system/DatasetSystemMetadataWriter.java

```diff
@@ -47,6 +48,7 @@
   public static final String EXPLORE_TAG = "explore";
   public static final String BATCH_TAG = "batch";
   public static final String TYPE = "type";
+  public static final String LOCAL_DATASET_TAG = "localds";
```

Comment:
Since users can view system tags should we have it more descriptive than `localds`? Maybe `local-dataset`?